src/test/java/api/support/fixtures/AgeToLostFixture.java

```diff
@@ -91,6 +92,16 @@ public AgeToLostResult createAgedToLostLoan(UnaryOperator<HoldingBuilder> holdin
     return ageToLostResult;
   }
 
+  public AgeToLostResult createLoanAgeToLostAndChargeFeesWithOverdues(IndividualResource lostPolicy, IndividualResource overduePolicy) {
+    
```

Comment:
Personally, I would remove this blank line as I don't think it enhances readability

Reply:
Removed.